### DISCUSSION TITLE
Consider adding value of NSLocalNetworkUsageDescription for Sunshine

### DISCUSSION THREAD

**Discussion by cathyjf** [LizardByte/community#549] [Sunshine] [answered]
This might help with LizardByte/Sunshine#3340. See also:
- https://developer.apple.com/documentation/technotes/tn3179-understanding-local-network-privacy#macOS-considerations
- https://developer.apple.com/documentation/bundleresources/information_property_list/nslocalnetworkusagedescription

**Comment by cathyjf** [comment]
Possibly helpful discussion on how to embed an `Info.plist` file into the binary without having to use an app bundle:
- https://discourse.cmake.org/t/how-to-embed-info-plist-in-a-simple-mac-binary-file/512/2

(Once LizardByte/community#546 is fixed, the `Info.plist` file will live separately in the bundle.)

**Comment by ReenigneArcher** [comment]
Converting to a feature request. You're free to make an attempt at this and create a PR.

**Comment by cathyjf** [comment]
I plan to make a pull request to address it. It's unclear why this isn't appropriate subject matter for a bug though.

**Comment by ReenigneArcher** [comment, marked as answer]
Added in https://github.com/LizardByte/Sunshine/pull/5186
